StyleCop.Analyzers/StyleCop.Analyzers/Lightup/.generated/Microsoft.CodeAnalysis.ResxSourceGenerator.CSharp/Microsoft.CodeAnalysis.ResxSourceGenerator.CSharp.CSharpResxGenerator/StyleCop.Analyzers.DocumentationRules.DocumentationResources.Designer.cs

```diff
@@ -0,0 +1,406 @@
+﻿// <auto-generated/>
+
+#nullable enable
+using System.Reflection;
+
+
+namespace StyleCop.Analyzers.DocumentationRules
+{
+    internal static partial class DocumentationResources
```

Comment:
💭 I'm leaning towards removing the generated output of ResxSourceGenerator from source control